src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -1300,4 +1299,28 @@ public Response syncExternalAccounts(@Context final HttpServletRequest httpServl
                     "Fatal error while attempting to synchronise users!", e).toResponse();
         }
     }
+
+    @POST
+    @Path("/start_quartz")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Start the Quartz Job Scheduler service if not already started.")
+    public Response resetMisuseMonitor(@Context final HttpServletRequest request) {
+        try {
+            RegisteredUserDTO user = userManager.getCurrentRegisteredUser(request);
+            if (!isUserAnAdmin(userManager, user)) {
+                return SegueErrorResponse.getIncorrectRoleResponse();
+            }
+            if (!segueJobService.isStarted()) {
+                segueJobService.initialiseService();
+                log.info(String.format("Admin user (%s) started Quartz scheduler successfully.", user.getEmail()));
+                return Response.ok(ImmutableMap.of("status", "Started successfully!")).build();
+            } else {
+                return Response.ok(ImmutableMap.of("status", "Already running.")).build();
```

Comment:
Do we know what happens if you try to start it and it is already running?
For this to be properly correct you'd want to hold a lock to make sure that separate requests don't ask if it is started between 1313 and 1314 or initialiseService() setting the isStarted flag. A `synchronized(segueJobService) {...}` block would be my suggestion assuming it is a singleton.

Reply:
What about making the actual method `public synchronized void initialiseService()` ?

Reply:
From the bytecode decompilation it looks like a no-op if you call start and it is started. This was only supposed to be for info, and not a serious indicator. But I agree about parallel requests and synchronising something to prevent that simultaneous init case.

Reply:
Making the initialiseService() synchronized will only make the requests queue up rather than not attempt it if it is not initialised or in the process of being initialised. You need the lock to cover checking and setting.
If it is a no-op it doesn't matter unless they decide to do something very strange in the future! This is unlikely to be a problem in practice.

Reply:
I now see that initialiseService() also checks whether or not it is already running, in that case synchronizing that method will do the trick 👍

Reply:
Yeah, I made the initialisation method idempotent so I also think this is all safe with regards to your concerns 🙌